src/modules/highlight/TextHighlighter.ts

```diff
@@ -817,7 +817,7 @@ export default class TextHighlighter {
           endOffset = selection.focusOffset;
         selection.collapse(selection.anchorNode, selection.anchorOffset);
 
-        var direction = [];
+        var direction: any[];
```

Comment:
I think declaring this as `let direction: string[] = [];` is the safest way to do this. Otherwise if you don't instantiate it, Typescript thinks it is definitely an array, but it isn't actually an array until it's instantiated... so if in the future we forget to instantiate it or something we'll run in to problems.

Reply:
For example I'm pretty sure typescript won't warn you about this:
```
var direction: any[];
direction.push("hi"); // ERROR because direction is still undefined, even though TS thinks it's an array
```

Reply:
i guess this happens when suggested autocomplete is accepted :) i'll change it tomorrow

Reply:
actually in this scenario, we never use it to push any values, since it's always initiated in the next few lines

Reply:
i could do this instead i guess:
```
        let direction = ["forward", "backward"];
        if (backwards) {
          direction = ["backward", "forward"];
        }

```